src/data_collection/create_map.py

```diff
@@ -0,0 +1,149 @@
+import ee
+import geemap.foliumap as geemap
+from geopy.geocoders import Nominatim
+from geopy.exc import GeopyError
+from decouple import config
+from constants import Cloud_Coverage, Spatial_Res, Geo_Tolerance, temperature_palette, temp_ranges
+
+HEAT_MAP_GEE_PROJECT = config("GEE_PROJECT")
+ee.Initialize(project=HEAT_MAP_GEE_PROJECT)
+
+
+def get_city_coordinates(city: str) -> list[int]:
+    """Asks user for city and convert to coordinates."""
+    city_name = city.title()
+    try:
+        geolocator = Nominatim(user_agent="urban_heat_island")
+        location = geolocator.geocode(city_name)
+
+    except GeopyError as e:
+        raise ValueError(f"Geolocation service error: {e}")
+
+    city_coordinates = [location.longitude, location.latitude]
```

Comment:
I would make this a named tuple object, then you can also make the type hint stricter